dev-support/docker/Dockerfile_aarch64

```diff
@@ -150,17 +152,6 @@ RUN pip3 install pylint==2.6.0 python-dateutil==2.8.1
 # hadolint ignore=DL3008
 RUN npm install -g bower@1.8.8
 
-###
```

Comment:
replaced the custom PhantomJS binary (which was built for Xenial and does not run on Bionic) with the official phantomjs package for Bionic.